src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final int DEFAULT_BOUND = 45;
+
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        switch (getRandomTypeOfFigures()) {
+            case CIRCLE :
+                return getRandomCircle();
+            case RECTANGLE :
+                return getRandomRectangle();
+            case RIGHT_TRIANGLE :
+                return getRandomRightTriangle();
+            case SQUARE :
+                return getRandomSquare();
+            default:
+                return getRandomIsoscelesTrapezoid();
+        }
+    }
+
+    public Figure getRandomIsoscelesTrapezoid() {
```

Comment:
why is this public and obvious? let's specify the private access modifier and order the methods correctly according to the modifiers if necessary